web/src/lib/llm/utils.ts

```diff
@@ -84,8 +84,31 @@ const MODEL_NAMES_SUPPORTING_IMAGES = [
 ];
 
 export function checkLLMSupportsImageInput(model: string) {
-  return MODEL_NAMES_SUPPORTING_IMAGES.some((modelName) => modelName === model);
+  return MODEL_NAMES_SUPPORTING_IMAGE_INPUT.some(
+    (modelName) => modelName === model
+  );
 }
+
```

Comment:
really thinking we should take the opportunity to attach capabilities to every model/provider in a single place, or even just loaded from a json file.